app/views/wpcc/your_details/new.html.erb

```diff
@@ -85,7 +85,7 @@
           </div>
         </div>
       </div>
-      <div class="details__row" data-dough-component="SalaryConditions">
+      <div class="details__row" data-dough-component="SalaryConditions" data-dough-salary-conditions-config="<%= ::Wpcc::MessagePresenter.opt_in_thresholds.to_json %>">
```

Comment:
same comment as above

Reply:
Done on the PR and in the commit message.